app/src/test/java/com/example/main_animal_villa/ExampleUnitTest.kt

```diff
@@ -11,7 +11,7 @@ import org.junit.Assert.*
  */
 class ExampleUnitTest {
     @Test
-    fun addition_isCorrect() {
+    fun additionISCorrect() {
```

Comment:
Camel case for naming functions per Kotlin convention https://kotlinlang.org/docs/coding-conventions.html#function-names

Reply:
In unit tests, using an _ to describe what is being tested, and actual results of that testing, is accepted and correct.  I'd leave this as it were.